app/templates/components/project-form.hbs

```diff
@@ -16,44 +14,87 @@
     {{input type="text" class='zap-project-id-field' value=model.zapProjectId}}
   </label>
 
-  <label class="text-small">Description
+  <label class="text-small">Project Description
     {{textarea class='zap-project-description-field' value=model.description rows="3"}}
   </label>
 
-  <h2 class="header-small">Project Geometries</h2>
-
   {{#if mapInstance}}
     {{#project-geometries-menu
       model=model
       mapInstance=mapInstance
       selectedLots=selectedLots
       lotSelectionMode=lotSelectionMode
       geometryMode=geometryMode
-      as |menu|}}
+      as |menu|
+    }}
+
+      {{!-- Development Site --}}
       {{menu.draw-control
-        mode="developmentSite"
-        modeDisplayName="Development Site"
+        mode='developmentSite'
+        modeDisplayName='Development Site'
         hasGeom=model.developmentSite
+        required=true
       }}
 
-      {{menu.draw-control
-        mode="projectArea"
-        modeDisplayName="Project Area"
-        hasGeom=model.projectArea
-      }}
+      {{!-- Project Area --}}
+      {{#if model.developmentSite}}
+        {{menu.draw-control
+          mode='projectArea'
+          modeDisplayName='Project Area'
+          hasGeom=model.projectArea
+        }}
+      {{/if}}
 
-      {{menu.draw-control
-        mode="rezoningArea"
-        modeDisplayName="Rezoning Area"
-        hasGeom=model.rezoningArea
-      }}
+      <hr>
+
+      {{#if model.proposedZoning}}
+        {{!-- Proposed Zoning --}}
+        {{menu.draw-control
+          mode='proposedZoning'
+          modeDisplayName='Zoning'
+          hasGeom=model.proposedZoning
+          resetAction=getClippedZoning
+        }}
+
+        {{!-- Proposed Commercial Overlays --}}
+        {{menu.draw-control
+          mode='proposedCommercialOverlays'
+          modeDisplayName='Commercial Overlays'
+          hasGeom=model.proposedCommercialOverlays
+          resetAction=getClippedCommercialOverlays
+        }}
+
+        {{!-- Proposed Special Purpose Districts --}}
+        {{menu.draw-control
+          mode='proposedSpecialPurposeDistricts'
+          modeDisplayName='Special Purpose Districts'
+          hasGeom=model.proposedSpecialPurposeDistricts
+          resetAction=getClippedSpecialPurposeDistricts
+        }}
+
+        {{!-- Rezoning Area --}}
+        {{menu.draw-control
+          mode='rezoningArea'
+          modeDisplayName='Rezoning Area'
+          hasGeom=model.rezoningArea
+        }}
```

Comment:
So cool to see this abstraction panning out and being helpful later down the road